src/DIRAC/Resources/MessageQueue/StompMQConnector.py

```diff
@@ -83,6 +86,7 @@ def setupConnection(self, parameters=None):
         connectionArgs = {
             "vhost": vhost,
             "keepalive": True,
+            "timeout": stompTimeout,
```

Comment:
Having tried it we also need something like `"heartbeats": (outgoingHeartbeatMs, incomingHeartbeatMs),` here as well. `15_000` seems like a sensible default